### ISSUE TITLE
CAIA: Update content in Veterans Crisis Line modal

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-cms#15731]
### Description
We need to update the TTY content in the Veterans Crisis Line modal to match the TTY information that's on the Veterans Crisis Line website (they no longer use the 800 number) and to make it consistent with our site style of removing "hearing loss." 

We need to replace the line that currently says:
Call TTY if you have hearing loss 800-799-4889

With this new content:

For TTY, call 711 then 988

### Time constraints / launch deadlines

No hard timing, though this has been outdated for a while so we would like to get this work in an upcoming sprint if possible. 

## Acceptance Criteria

- [x] CAIA signoff in Review instance prior to merge
- [x] A11y review pre-merge
- [x] Content is updated in the VCL on VA.gov to change "Call TTY if you have hearing loss 800-799-4889" to "[For TTY, call 711 then 988]" as one link, with 711 as the `tel:` attribute
  - [x] in desktop header
  - [x] in the mobile header
  - [x] in the injected header

**Comment by jilladams** [comment]
@RLHecht can you clarify how https://www.veteranscrisisline.net/ handles it? FE eng would like to understand what portions of the updated copy should be a link, but we can't find an example in the site.

What we see here, https://www.veteranscrisisline.net/get-help-now/call/ is , `For TTY users: Use your preferred relay service or dial 711 then 988.`

**Comment by jilladams** [comment]
From refinement: 
1. We will update the new text here, with RandiH's notes about links
2. We will consult with Laura to make sure it has the right aria labels and a11y notes
3. Laura will ticket an a11y review of the VCL modal to address other issues like: using <va-modal> component, and aria labels, etc. on other text / links.

**Comment by RLHecht** [comment]
@jilladams We would want to link the whole line: [For TTY, call 711 then 988] to have it match the other bullets in that modal.

**Comment by jilladams** [comment]
@RLHecht Thanks. Right now, for the current "988 ext 1" cta, the action just dials 988. We dont' handle the ext 1 or pausing, etc. Does that expectation line up here, that we would be making 711 the action, and then callers would be expected to handle entering 988?

**Comment by RLHecht** [comment]
@jilladams Yes, that seems right. Thanks!

**Comment by chriskim2311** [comment]
Code is complete, waiting on PR review.

**Comment by chriskim2311** [comment]
@RLHecht Hi! Code has been completed for this change to the crisis modal. Can you take a look to verify the content thank you!

@laflannery Hi! Can you also take a look at the new changes thank you!

Review Instance: http://c9c02912609cab055b2d41845b796981.review.vetsgov-internal/

**Comment by chriskim2311** [comment]
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeGx3QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4711fd561503cdbf58b511af04ac4b0c711dcab8/VA_gov___Veterans_Affairs.png)

**Comment by laflannery** [comment]
@chriskim2311 I just have one edit, can you remove the + from the `href`? I double checked and I don't believe this should be here, we don't have it on the contact page:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/0ce86b3c-bfa4-46ba-99f9-744648265790)

**Comment by chriskim2311** [comment]
@laflannery Done!

**Comment by randimays** [comment]
@jilladams @chriskim2311 This may in a Slack message somewhere that I haven't gotten to yet: have we noted somewhere the style differences between the va.gov crisis line modal and the injected header crisis line modal (spacing and link color & underline)?

I'm not suggesting we expand the scope of this ticket. But we could ticket it **if** we think this could be worth addressing before the redesign of the header/footer.

va.gov
<img width="550" alt="Screenshot 2023-10-30 at 8 29 07 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/2bcd0097-f4b1-4a41-9339-489d66d4194d">

injected header
<img width="550" alt="Screenshot 2023-10-30 at 8 29 14 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/7f0936b0-43b8-4fd6-8992-aa6caab3242b">

**Comment by jilladams** [comment]
Not that i know of! Are the styles in the injected header easy (HA) to get ahold of and align with what happens on VA.gov? AKA: what's the additional lift to Just Do It?  If we're not sure or we know it's not easy, we can whip up a backlog ticket. If it _is_ easy, I support you going for it under this cover.

**Comment by FranECross** [comment]
@randimays @chriskim2311 Just chiming in to agree with what Jill said. 😁☝🏼Thanks!

**Comment by randimays** [comment]
@chriskim2311 Do you have the bandwidth to tackle this modal styling as part of this ticket? If not we can add a new ticket for later.

**Comment by chriskim2311** [comment]
@RLHecht Hi! I wanted to know if Megan got a chance to review this. PR is ready to merge if she signs off thanks!

**Comment by megzehn** [comment]
This looks good to me!! Thank you @chriskim2311 and @RLHecht!

**Comment by chriskim2311** [comment]
@megzehn @RLHecht @jilladams @FranECross @randimays 
Thanks for the collaboration on this ticket! I will be merging today if there are no objections.

**Comment by chriskim2311** [comment]
This has not made it to production yet. There has been some unit test issues that were blocking releases for vets-website. I will have this out on Monday!

**Comment by jilladams** [comment]
This is in prod and verified on desktop, mobile, injected headers.
